2019/python/rules_engine.py

```diff
@@ -2416,7 +2416,7 @@ def v696_3(self):
 		fields_1 = ["aus_1", "aus_2", "aus_3", "aus_4", "aus_5"]
 		fields_2 = ["aus_result_1", "aus_result_2", "aus_result_3", "aus_result_4", "aus_result_5"]
 		vals_1 = ("1", "2", "3", "4", "5", "6")
```

Comment:
1111 is a valid code for AUS result 1, so we can't take it out of this check.

I think you need to add 1111 to the values set for AUS1 here as well.

Reply:
Yep - adding the exemption code back in worked. Thanks!